semantic/src/themes/universe/elements/button.overrides

```diff
@@ -25,6 +25,12 @@
   background-color: @coralDarker;
 }
 
+.ui.buttons .inverted.primary.button:hover:enabled {
+  box-shadow: 0 0 0 1px @blue inset !important;
+  background-color: inherit;
+  color: @blueDark;
+}
```

Comment:
Hmm isn't the design feedback more specific than this? 
I don't think we want to remove the hover on all primary buttons, I think we only want to remove the hover on the active/selected button of a button group? 
![image](https://user-images.githubusercontent.com/22333438/58589794-b51b0a00-8230-11e9-86ab-6e4e8f9b5fe7.png)

Reply:
yup, doesn't `.ui.buttons` cover the button _group_ aspect of it?

Reply:
True! yeah i think it would